core/statistics/tpsBenchmark_test.go

```diff
@@ -309,6 +309,8 @@ func TestTpsBenchmark_EmptyBlocksShouldNotUpdateMultipleTimes(t *testing.T) {
 }
 
 func TestTpsBenchmark_Concurrent(t *testing.T) {
+	t.Skip("this test is failing. Task EN-3206")
```

Comment:
add a TODO here as well.

Reply:
+1

Reply:
Task 3206 was merged. I will remove this line and run the tests to see if it's fixed